### PR TITLE
Remove Construct UI components from search bar and refactor

### PR DESCRIPTION
**Pull request by affordances**
**Punting:** 

- filtering
- history
- chain-specific behavior
- custom domain behavior

Screenshot 2022-12-20 at 6 00 25 PM

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/pages/search/index.tsx [hunk 1]
```diff
@@ -147,21 +147,21 @@ const getCommentResult = (comment, searchTerm) => {
...
-    onclick: () => {
-      m.route.set(
-        `/${chain}/proposal/${proposalId.split('_')[0]}/${
-          proposalId.split('_')[1]
-        }`
-      );
-    },
+    // onclick: () => {
+    //   m.route.set(
+    //     `/${chain}/proposal/${proposalId.split('_')[0]}/${
```
Comment (zakhap) [inline packages/commonwealth/client/scripts/views/pages/search/index.tsx:152]: is there a reason this and `onSelect` above are commented out?

File: packages/commonwealth/client/scripts/views/pages/search/search_bar.tsx [hunk 2]
```diff
@@ -89,16 +89,16 @@ const getCommunityPreview = (
...
-          : params.community.id
-          ? `/${params.community.id}`
-          : '/'
-      );
-    }
-    closeResultsFn();
+    // if (params.community) {
+    //   m.route.set(
+    //     params.community.address
+    //       ? `/${params.community.address}`
```
Comment (zakhap) [inline packages/commonwealth/client/scripts/views/pages/search/search_bar.tsx:95]: same here?

**Review by dillchen — COMMENTED** [review]
LGTM (clicked across all search states between 2 - 3 communities), should get another review from @zakhap

File: packages/commonwealth/client/scripts/views/pages/search/index.tsx (hunk 1 above)
Comment (zakhap) [reply packages/commonwealth/client/scripts/views/pages/search/index.tsx:152]: fixed this

**Review by zakhap — APPROVED** [review]

File: packages/commonwealth/client/scripts/views/pages/search/search_bar.tsx (hunk 2 above)
Comment (zakhap) [reply packages/commonwealth/client/scripts/views/pages/search/search_bar.tsx]: fixed

**Pull request merged**